### PR TITLE
Fixes Vamp Passives (#2812)

### PR DESCRIPTION
**Pull request by VampyrBytes**
Fixes Vampire thermal vision
Improved regen was working, but the message was commented out due to
spam, it now shows without spamming

### CODE REVIEW

**Comment by Regen1** [comment]
I was improved?

**Comment by VampyrBytes** [comment]
Lol :smile:

File: code/game/gamemodes/vampire/vampire.dm [hunk 1]
```diff
@@ -324,8 +325,9 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 			vamp.powers.Add(VAMP_BATS)
 		if(!(VAMP_SCREAM in vamp.powers))
 			vamp.powers.Add(VAMP_SCREAM)
-		// Commented out until we can figured out a way to stop this from spamming.
-		//src << "\blue Your rejuvination abilities have improved and will now heal you over time when used."
+		if(!(vamp.upgradedRegen))	// to prevent spamming
+			src << "<span class='notice'>Your rejuvination abilities have improved and will now heal you over time when used."
```
Comment (Markolie) [inline code/game/gamemodes/vampire/vampire.dm]: The <span> isn't closed here.

Comment (VampyrBytes) [inline code/game/gamemodes/vampire/vampire.dm]: Good catch. PR's up to fix it

**Pull request merged**